### PR TITLE
feat(web): add last login column to team users table

### PR DESCRIPTION
**Pull request by robot9706**
Adds a last login column to the users table on the teams page.

### CODE REVIEW

**Comment by bvektor** [comment]
[next-translate] "teams:lastLogin" is missing in current namespace configuration. Try adding "lastLogin" to the namespace "teams".

File: web/crux/src/app/team/team.mapper.ts [hunk 1]
```diff
@@ -23,12 +31,19 @@ export class TeamMapper {
       return null
     }
 
+    const userSessions = sessions.get(user.userId)
+    const lastSession = userSessions
+      .filter(it => it.authenticated_at !== undefined)
```
Comment (m8vago) [inline web/crux/src/app/team/team.mapper.ts:36]: `!!it.authenticated_at`

Comment (polaroi8d) [reply web/crux/src/app/team/team.mapper.ts:36]: Nice catch my bro.

**Review by polaroi8d — APPROVED** [review]
LGTM, you can merge after rebase.

**Review by m8vago — APPROVED** [review]

**Pull request merged**